docs/docs/guides/solution-scenarios/configurations.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: How to configure ??
+---
+
+Each customer does have different needs and use-cases. In ZITADEL you are able to configure your settings depending on your needs.
+In this capital we show you different use-cases we have already seen, that could interest you.
+
+
+## Automatically redirect if organization has one identity provider
+
+You have different organizations/customers in your ZITADEL instance and they have different needs on how to authenticate. One of your customers does only allow login with an external identity provider like Google, Azure AD, etc.
+If a user of this organization wants to login, you don't want them to enter the username in the ZITADEL Login UI, they should be redirected directly to the identity provider.
+
+## Settings
+
+1. Go to the "Identity Providers" Settings of your organization
+2. Configure your needed identity provider: Read this [guide](../integrate/identity-brokering.md) if you don't know how
+3. Go to the "Login Behavior and Security" settings of the organization
+4. Disable "Username Password Allowed" and enable "External IDP allowed" in the Advanced Section
+
+
+Send either the the organization id (`urn:zitadel:iam:org:id:{id}`) or organization primary domain (`urn:zitadel:iam:org:domain:primary:{domainname}`) scope on your authentication request from your application, to identify on which organization the users should login.
+
+More about the [scopes](../../apis/openidoauth/scopes#reserved-scopes)
+
+
+## Custom app domains per organization
+
+I have an application that runs on different domains for each customer I have. When the user wants to login I redirect to ZITADEL and want to get back to my application after. This should still be the custom domain for my organization.
+
+Example:
+MyApplication: customer-a.app.com
+ZITADEL Login: login.app.com
+
+In the OIDC Authorization request you always have to send the redirect url to where you like to be redirected after login.
+It is possible to register multiple urls on one application, the only criteria is that the requested url has to match one of the registered urls.
+
+Read more about [applications](../manage/console/applications) and the [redirect urls](../manage/console/applications#redirect-uris)
+
+### Trigger organization in ZITADEL login
```

Comment:
```suggestion
## Trigger a specific Organization from you Application
```

Reply:
This kind of feels redundant to the first section, or am I missing something?

Reply:
Yes it is. But the first section has nothing to do with the second. So we need this part in both sections.